omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -287,6 +287,9 @@
 
 default['private_chef']['elasticsearch']['enable'] = false
 default['private_chef']['elasticsearch']['first_internal_install'] = false
+default['private_chef']['elasticsearch']['internal_install'] = false
+default['private_chef']['elasticsearch']['install_major_version'] = 6
```

Comment:
we might not need this at all. since we know the version internal es and chef-backend es re on and external es will be able to respond to the  rest get.

Reply:
It might be simpler to have only one path.